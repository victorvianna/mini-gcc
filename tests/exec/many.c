int many(int a, int b, int c, int d, int e, int f, int g, int h, int i, int j) {
  putchar(64 + a);
  putchar(64 + b);
  putchar(64 + c);
  putchar(64 + d);
  putchar(64 + e);
  putchar(64 + f);
  putchar(64 + g);
  putchar(64 + h);
  putchar(64 + i);
  putchar(64 + j);
  putchar(10);
  if (a < 10)
    many(b, c, d, e, f, g, h, i, j, a);
  return 0;
}
int main() {
  many(1, 2, 3, 4, 5, 6, 7, 8, 9, 10);
  return 0;
}
