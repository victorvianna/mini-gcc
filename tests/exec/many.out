ABCDEFGHIJ
BCDEFGHIJA
CDEFGHIJAB
DEFGHIJABC
EFGHIJABCD
FGHIJABCDE
GHIJABCDEF
HIJABCDEFG
IJABCDEFGH
JABCDEFGHI
